src/Network/Receive.pm

```diff
@@ -3844,8 +3844,8 @@ sub npc_talk_close {
 		$messageSender->sendTalkCancel($ID);
 	}
 
-	$ai_v{npc_talk}{talk} = 'close';
-	$ai_v{npc_talk}{time} = time;
+	$ai_v{'npc_talk'}{'talk'} = 'close';
+	$ai_v{'npc_talk'}{'time'} = time;
```

Comment:
Please next time do style changes in a separate commit, since it gets in the way of review in other places where style changes interleave with actual changes.